services/download/src/main/resources/bootstrap-cloud.yaml

```diff
@@ -5,7 +5,7 @@ spring:
   cloud:
     vault:
       ssl:
-        trust-store: file:${SSL_VAULT_TRUSTSTORE_PATH}
```

Comment:
Are we sure the truststore  is not injected in the container file system?

Reply:
I did this because vault ssl was not working on dev when using the prefix "file:". Maybe we need to investigate this